index.js

```diff
@@ -620,6 +620,26 @@ function fastifyMultipart (fastify, options, done) {
   done()
 }
 
+/**
+ * Adds a new type `isFile` to help @fastify/swagger generate the correct schema.
+ */
+function ajvFilePlugin (ajv) {
+  return ajv.addKeyword({
+    keyword: 'isFile',
+    compile: (_schema, parent) => {
+      // Updates the schema to match the file type
+      parent.type = 'string'
```

Comment:
Note that changing the actual schema here will have side-effect.
Not sure how it impact but let see what the community feedback.

Reply:
What kind of side effect are you talking about? [binary string is what is specified](https://swagger.io/docs/specification/describing-request-body/file-upload/).

Reply:
You are modifying the user input and we don't know how it's behave when he / she re-use such modified schema.
That is what side-effect means.